server/api/boundaries/poamDownload.post.ts

```diff
@@ -2,14 +2,17 @@ import intoStream from "into-stream";
 import { sendStream } from "h3";
 import jwt from "jsonwebtoken";
 import { generatePoam } from "../../utils/excelExport/poamExport";
+import { User, Boundary, BoundaryInterface } from "../../../db/models";
 
 export default defineEventHandler(async (event) => {
   const rawToken = getCookie(event, "tirtoken");
   const config = useRuntimeConfig();
   const decodedToken = jwt.verify(rawToken, config.jwt_key) as { [key: string]: any };
 
   const userId = decodedToken.userId;
+  const user = await User.findByPk(userId);
```

Comment:
If we need to do the userid validation, then we should do it here before we look up the user.